integration-tests/modules/__tests__/promotion/admin/promotion-rules.spec.ts

```diff
@@ -342,6 +342,194 @@ medusaIntegrationTestRunner({
           )
```

Comment:
comment(this file): At some point, this file should be moved to `integration-tests/api`

Reply:
Shouldn't we wait for the CI to run v2 api before doing that?

Reply:
Yeah, that's true